dojo/tools/harbor_vulnerability/parser.py

```diff
@@ -0,0 +1,92 @@
+import json
+from dojo.models import Finding
+
+
+class HarborVulnerabilityParser(object):
+
+    """
+    Read JSON data from Harbor compatible format and import it to DefectDojo
+    """
+
+    def __init__(self, filename, test):
+
+        self.items = []
+        if filename is None:
+            return
+
+        tree = filename.read()
+        try:
+            data = json.loads(str(tree, "utf-8"))
+        except:
+            data = json.loads(tree)
+
+        # When doing dictionary, we can detect duplications
+        dupes = dict()
+
+        vulnerability = data[
+            "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
```

Comment:
hello @natebwangsut ,
What if there is an update in version. it will be better to use something like this
`data[data.keys()[0]]['vulnerabilities']`
thanks

Reply:
I agreed with that recommendation, i'll make a change to the PR